livro/inc/gutenberg-dependency-check.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+add_action( 'admin_notices', 'show_admin_messages' );
+
+function show_admin_messages() {
+	$metadata = file_get_contents( get_stylesheet_directory() . '/style.css' );
+	preg_match( '/(?<=Requires Gutenberg:).+/', $metadata, $match );
+	if ( 0 === sizeof( $match ) ) {
+		return;
+	}
+	$version = $match[0];
+	if ( is_plugin_active( 'gutenberg/gutenberg.php' ) ) {
```

Comment:
this naming convention will not be true for all environments, as it's certainly not the case over on dotcom

Reply:
Yea, I was afraid of that.  I was hoping I could check for activation in the loop above but it just tells what's installed, not what's activated.  Is there a better way to tell if a plugin (generically) is activated or should we do something Gutenberg specific?

Reply:
I don't know! I think I tried something like this before for similar reasons and got stuck at this point too.

Reply:
https://developer.wordpress.org/reference/functions/is_plugin_active/

Reply:
Maybe we can use `class_exists`?

Reply:
I found [this](https://github.com/WordPress/gutenberg/blob/trunk/lib/load.php#L12) which seems to be suited to the purpose.

Checking for that constant seems to work well.